src/cmd/ksh93/tests/basic.sh

```diff
@@ -832,5 +832,21 @@ got="$(join <(printf '%d\n' 1 2) <(printf '%d\n' 1 2))"
 [[ $exp == $got ]] || err_exit "pipeline fails with illegal seek error" \
 	"(expected $(printf %q "$exp"), got $(printf %q "$got"))"
 
+# ======
+# In ksh93v- and ksh2020 eval'ing a function definition may dump the
+# function body to stdout.
+# https://github.com/att/ast/issues/1160
+got="$($SHELL -c '
+	for i in $(seq 1024)
```

Comment:
A few things on this one:
* The `seq` command is non-standard and unportable. Please use an arithmetic `for` loop instead.
* I think this test would be better placed in functions.sh.
* I can't get this test to fail on my macOS system when running it on 93v- or ksh2020 (using `bin/shtests -p basic SHELL=/usr/local/bin/ksh2020`, etc.).